### PR TITLE
Also use LANG env var to check for UTF-8 output support

### PR DESCRIPTION
**Pull request by GustavoLCR**
* Also boolify r_cons_is_utf8()

### CODE REVIEW

**Comment by radare** [comment]
see comment in the other pr

**Comment by radare** [comment]
also https://github.com/radareorg/radare2/pull/15256#issuecomment-541282197

**Comment by XVilka** [comment]
What was decided? Can you please rebase to make it green? I think we should merge it.

File: libr/cons/utf8.c [hunk 1]
```diff
@@ -191,40 +191,47 @@ static int cursor_position(const int tty, int *const rowptr, int *const colptr)
...
 	char *sval = r_sys_getenv ("LC_CTYPE");
 	if (sval) {
 		r_str_case (sval, 0);
 		if (!strcmp (sval, "utf-8")) {
-			ret = 1;
+			ret = true;
 		}
 		free (sval);
 	}
+	char *lang = r_sys_getenv ("LANG");
```
Comment (radare) [inline libr/cons/utf8.c]: this conflicts with scr.utf8 when spawning a new process. this is why the tests are failing

**Comment by XVilka** [comment]
Should be fine after https://github.com/radareorg/radare2/pull/15273 is merged I think.

**Comment by GustavoLCR** [comment]
This one doesn't affect windows.

**Pull request merged**